packages/apps/spaces/ui/form/Button/Button.variants.ts

```diff
@@ -2396,12 +2397,13 @@ export const outlineButton = cva(
         ],
         error: [
           'text-error-700',
+          'bg-error-50',
           'border',
           'border-solid',
           'border-error-300',
-          'hover:bg-error-50',
+          'hover:bg-error-100',
           'hover:text-error-700',
-          'focus:bg-error-50',
+          'focus:bg-error-100',
         ],
         rose: [
```

Comment:
THis file is generated through a script based on the color schemes we have. it should not be edited manually. We need to change the variant template which i think it's weird since we've just triple checked all colors for those variants from figma.

Reply:
What you can do instead if you need special color for this usecase, which againm I would argue against, is adding a className on your button instance and add those colors as overrides.

Reply:
```ts
<button className='bg-error-50 hover:bg-error-100 focus:bg-error-100' />
```

Reply:
this is standard variant, updated codegen